internal/factory/draft_pr_test.go

```
@@ -0,0 +1,244 @@
+	_, err = service.CreateDraftPullRequest(context.Background(), factory.DraftPullRequestRequest{RunID: "run-active"})
+	if err == nil || !strings.Contains(err.Error(), "still has active implementation invocation") {
+		t.Fatalf("CreateDraftPullRequest() error = %v, want validation before Git effects", err)
+	}
+}
+
+// draftGitWorkspace records the host Git effects for coordinator seam tests.
+type draftGitWorkspace struct {
+	state         gitadapter.WorktreeState
+	checkpoints   []gitadapter.CheckpointRequest
+	pushes        []gitadapter.PushRequest
+	checkpointSHA string
```

golangci-lint (2.12.2)

[error] field checkpointSHA is unused

(unused)